src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -359,6 +366,17 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
+		}
+
+        private void InitializeNamespacesToExcludeFromLibrary()
+        {
+            if (!NamespacesToExcludeFromLibrarySpecified)
```

Comment:
The boolean value will (automatically) return true if the tag `<NamespacesToExcludeFromLibrary>` is found in the XML, whether it is empty or having some other elements.